front/app/containers/Admin/projects/components/ProjectRow/ProjectMoreActionsMenu.tsx

```diff
@@ -32,47 +35,38 @@ const ProjectMoreActionsMenu = ({ projectId, setError }: Props) => {
       setError(error);
     }
   };
-
-  const copyAction = {
-    handler: async () => {
-      setIsCopying(true);
-      await handleCallbackError(
-        () => copyProject(projectId),
-        formatMessage(messages.copyProjectError)
-      );
-      setIsCopying(false);
-    },
-    label: formatMessage(messages.copyProjectButton),
-    icon: 'copy' as const,
-    isLoading: isCopying,
-  };
-
-  const deleteAction = {
-    handler: async () => {
-      if (window.confirm(formatMessage(messages.deleteProjectConfirmation))) {
-        setIsDeleting(true);
+  const actions: IAction[] = [
+    {
+      handler: async () => {
+        setIsCopying(true);
         await handleCallbackError(
-          () => deleteProject(projectId),
-          formatMessage(messages.deleteProjectError)
+          () => copyProject(projectId),
+          formatMessage(messages.copyProjectError)
         );
-        setIsDeleting(false);
-      }
+        setIsCopying(false);
+      },
+      label: formatMessage(messages.copyProjectButton),
+      icon: 'copy' as const,
+      isLoading: isCopying,
     },
-    label: formatMessage(messages.deleteProjectButtonFull),
-    icon: 'delete' as const,
-    isLoading: isDeleting,
-  };
-
-  const actions: IAction[] = [];
-  const canModerate = canModerateProject(projectId, { data: authUser });
-  const isAdminUser = isAdmin({ data: authUser });
-
-  if (isAdminUser || canModerate) {
-    actions.push(copyAction);
-  }
+  ];
```

Comment:
Component is now only rendered if we have permissions so the copy project action can be there by default.